packages/module/src/LogSnippet/LogSnippet.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { CodeBlock, CodeBlockAction, CodeBlockCode, ClipboardCopyButton, Button, ClipboardCopy } from '@patternfly/react-core';
+
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  logSnippet: {
```

Comment:
We usually use clsx with this.  Is there a reason we aren't here?  I think we should use a layout instead of the div and p below.

Reply:
I thought clsx was just for conditional rendering